pandas/tests/test_config.py

```diff
@@ -428,3 +428,7 @@ def test_option_context_scope(self):
 
         # Ensure the current context is reset
         assert self.cf.get_option(option_name) == original_value
+
+    def test_dictwrapper_getattr(self):
+        options = self.cf.options
```

Comment:
use an with pytest.raises here to check for the OptionError

Reply:
As well as the hasattr test or instead of it? hasattr already tests for OptionError.